jumpstarter/config/exporter.py

```diff
@@ -29,19 +31,52 @@ def instantiate(self) -> Driver:
 class ExporterConfigV1Alpha1(BaseModel):
     BASE_PATH: ClassVar[Path] = Path("/etc/jumpstarter/exporters")
 
-    apiVersion: Literal["jumpstarter.dev/v1alpha1"]
-    kind: Literal["ExporterConfig"]
+    alias: str = Field(default="default", exclude=True)
+
+    apiVersion: Literal["jumpstarter.dev/v1alpha1"] = "jumpstarter.dev/v1alpha1"
+    kind: Literal["ExporterConfig"] = "ExporterConfig"
+    metadata: Metadata
 
     endpoint: str
     token: str
 
-    export: dict[str, ExporterConfigV1Alpha1DriverInstance]
+    export: dict[str, ExporterConfigV1Alpha1DriverInstance] = Field(default_factory=dict)
+
+    @property
+    def path(self):
+        return self.__path(self.alias)
+
+    @classmethod
+    def __path(cls, alias: str):
+        return (cls.BASE_PATH / alias).with_suffix(".yaml")
 
     @classmethod
-    def load(cls, name: str):
-        path = (cls.BASE_PATH / name).with_suffix(".yaml")
+    def load_path(cls, path: Path):
         with path.open() as f:
-            return cls.model_validate(yaml.safe_load(f))
+            config = cls.model_validate(yaml.safe_load(f))
+            return config
+
+    @classmethod
+    def load(cls, alias: str):
+        config = cls.load_path(cls.__path(alias))
+        config.alias = alias
+        return config
+
+    @classmethod
+    def list(cls):
+        exporters = []
+        with suppress(FileNotFoundError):
+            for entry in cls.BASE_PATH.iterdir():
+                exporters.append(cls.load(entry.stem))
+        return exporters
+
+    def save(self, delete=False):
```

Comment:
```suggestion
    def save(self, delete=False):
```

Why not just having a delete method?